Detectors/MUON/MCH/Evaluation/README.md

```diff
@@ -17,3 +17,40 @@ $ o2-mch-tracks-reader-workflow --hbfutils-config o2_tfidinfo.root --infile mcht
 $ root
 root[0] o2::mch::eval::drawAll("compare.root")
 ```
+# MCH Cluster Maps
+
+The general purpose is to track "unexpected" detector issues not well reproduced with MC simulations. These problems generate non-negligible bias in Acc*Eff corrections resulting in large tracking systematic uncertainties.
+
+During the data reconstruction, the status of the detector is calculated with the CCDB which is used to discard most of the detector issues.
+This status map is built with information based on pedestals, high voltage tension, occupancy etc.
```

Comment:
high voltage is not currently included in the computation of the statusmap. It's planned, but not yet done.